jetson/nav/gate_search/gateStateMachine.cpp

```diff
@@ -288,24 +294,24 @@ NavState GateStateMachine::executeGateDriveToFarPost()
     return NavState::GateDriveToFarPost;
 } // executeGateDriveToFarPost()
 
-// Turn back to center point
+// Execute turn back to center point for driving through the gate
 NavState GateStateMachine::executeGateTurnToGateCenter()
 {
     if ( mPhoebe->turn( centerPoint2 )) 
     {
         return NavState::GateDriveThrough;
     }
-
     return NavState::GateTurnToGateCenter;
 } // executeGateTurnToGateCenter()
 
+// Drive through gate posts
 NavState GateStateMachine::executeGateDriveThrough()
 {
     DriveStatus driveStatus = mPhoebe->drive( centerPoint2 );
 
     if( driveStatus == DriveStatus::Arrived )
     {
-        if(!CP1ToCP2CorrectDir)
+        if(!CP1ToCP2CorrectDir) // Check if we drove through the incorrect direction
```

Comment:
Make spacing consistent with the rest of `jetson/nav` code. I.e., `if( !CP1ToCP2CorrectDir )`.

Please do this throughout.